cloudsdk/src/main/java/io/particle/android/sdk/cloud/models/SignUpInfo.java

```diff
@@ -0,0 +1,91 @@
+package io.particle.android.sdk.cloud.models;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import com.google.gson.annotations.SerializedName;
+
+import javax.annotation.Nullable;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+/**
+ * Required and optional user information used in sign up process.
+ */
+@ParametersAreNonnullByDefault
+public class SignUpInfo implements Parcelable {
+    private String username, password;
```

Comment:
Inlining multiple field declarations like this makes them easy to miss when skimming code, IMO.

I'm not going to block the PR approval on it, but I don't think it's worth the tradeoff.

(Same goes for the annotations below)